### DISCUSSION TITLE
FIL Dev Summits - Singapore & Iceland

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#655] [Developers]
## FIL Dev Summits - Singapore & Iceland

The Filecoin Developer Summit (FIL Dev Summit) gathers developers, builders, and community members eager to contribute to the core protocol and network evolution of Filecoin. This is the perfect place to bring in complex/challenging problems and opportunities, brainstorm solutions collectively, and build collaborative connections between teams! 

## Who is the FIL Dev Summit For?

- Core protocol developers
- Implementation developers
- Advanced  data onboarding tooling architects
- Filecoin application/tooling developers (FVM & IPC)
- Retrieval market developers
- Storage provider/service developers
- Large storage clientele
- Governance-engaged token holders
- Filecoin Improvement Proposal (FIP) contributors

## Two Part Event

The event has two parts, from Asia (Singapore) to Europe/NA (Iceland is geographically in between!). We will kick off the deeper conversations about protocol trajectory, challenges, and opportunities in Part 1, and refine collectively into Part 2 with more discussion, presentations, and alignments. 

### Part 1: FILDev Singapore 🇸🇬 - 12th to 14th September 2023

🇸🇬 Protocol Development & Evolution: Part 1

🇸🇬 IPC & friends

🇸🇬 FVM application & tooling

🇸🇬 SP tooling & stack

🇸🇬 Client Data Onboarding protocol/tools/market Part 1

🇸🇬 Governance & Community Part 1

### Part 2: FILDev Iceland 🇮🇸 - 25th to 27th September 2023

🇮🇸 Protocol Development & Evolution: Part 2

🇮🇸 Filecoin Retrievals & Data Availability

🇮🇸 Scalability & compute over data

🇮🇸 FVM & Runtimes

🇮🇸 Client Data Onboarding protocol/tools/market Part 2

🇮🇸 Clients’ Onboarding Experience

🇮🇸 Chain tooling & infrastructure

🇮🇸 Governance & Community Part 2

### Call to Action

- 🎫 Wanna participate in the summit? Apply to attend!
- 🗣️ Wanna be a part of the tracks? Submit a talk/demo/session!
- 🔍 The topic you want to share/discuss not included yet? Submit a track!
- ⭐ Wanna help organize/sponsor the first FILDev Summit? Contact us at [[#fil-dev-summit](https://filecoinproject.slack.com/archives/C05LHC1ULG1)](https://filecoinproject.slack.com/archives/C05LHC1ULG1)!
- 🪶 Spread the news! More details at [https://fildev.io/](https://fildev.io/).

## FILDev Links 🖇️

🔗 [[Official events website - fildev.io](https://fildev.io/)](https://fildev.io/)

🇸🇬 Singapore - [[Submit a Talk/Track](https://airtable.com/appEjnh5rpWMsjocb/shrw3Ha0yTusDmcOg)](https://airtable.com/appEjnh5rpWMsjocb/shrw3Ha0yTusDmcOg)

🇸🇬 Singapore - [[Apply to Attend](https://airtable.com/appEjnh5rpWMsjocb/shran7li4slfQSB04)](https://airtable.com/appEjnh5rpWMsjocb/shran7li4slfQSB04)

🇮🇸 Iceland - [[Submit a Talk/Track](https://airtable.com/appEjnh5rpWMsjocb/shr6SmQjqdgn5Pc90)](https://airtable.com/appEjnh5rpWMsjocb/shr6SmQjqdgn5Pc90)

🇮🇸 Iceland - [[Apply to Attend](https://airtable.com/appEjnh5rpWMsjocb/shran7li4slfQSB04)](https://airtable.com/appEjnh5rpWMsjocb/shran7li4slfQSB04)

🔗 [[Apply to sponsor](https://airtable.com/appEjnh5rpWMsjocb/shrn57n37JyLSOTlV)](https://airtable.com/appEjnh5rpWMsjocb/shrn57n37JyLSOTlV)

**Comment by wolfduan** [comment]
github
